resources/js/composables/usePasswordVisibility.ts

```diff
@@ -0,0 +1,6 @@
+import { ref } from 'vue'
+
+export const showPasswords = ref(false)
+export function togglePasswords() {
+  showPasswords.value = !showPasswords.value
+}
```

Comment:
This is not a composable. It should look like this, for example:


```suggestion
export const useTooglePassoword = () => {
    const showPasswords = ref(false)
    const togglePasswords = () => {
      showPasswords.value = !showPasswords.value
    }
    
    return {
        showPasswords,
        togglePasswords,
    }
} 
```